python/sqlalchemy/README.md

```diff
@@ -0,0 +1,50 @@
+# Yugabyte Python ORM implementation sample using sqlalchemy
+
+This example project shows you how you can use Python with sqlalchemy to connect to a Yugabyte cluster and perform various ORM operations.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+
+### Prerequisites
+
+* Yugabyte
+* Python3
+* sqlalchemy
+
+### Installing
```

Comment:
Move this above line 9, and change to a second level header (## instead of ###)

Reply:
Done